.teamcity/Gradle_Check/model/CIBuildModel.kt

```diff
@@ -218,7 +218,7 @@ data class TestCoverage(val testType: TestType, val os: OS, val testJvmVersion:
     }
 
     fun suffix(): String {
-        return if(buildJvmVersion != JvmVersion.java8) "_${buildJvmVersion.name.capitalize()}" else ""
+        return ""
```

Comment:
Previously I used this to test if TC is Java 9 ready. Basically, when a configuration runs on Java9, its name would be `<OriginalConfigurationName>_Java9` to avoid name conflict with other configurations.

Reply:
Let's remove this method completely, shall we? If we ever need to test this again, I suggest we create a whole new root project (analogous to `Check`) which then sets the default Java version to something else.